src/node.cc

```diff
@@ -3129,6 +3129,7 @@ void LoadEnvironment(Environment* env) {
   // thrown during process startup.
   try_catch.SetVerbose(true);
 
+  setvbuf(stderr , NULL , _IOLBF , 1024);
```

Comment:
On a separate testing branch, I moved it to `main()`.

I had it run the test on SmartOS 500 times and there were no failures. https://ci.nodejs.org/job/node-test-commit-smartos/246/

Then I commented out the line.
https://github.com/nodejs/node/commit/bf88e85653bc27ee8ef8c5a26b54dcff7d641851

And ran the test again and got multiple failures.
https://ci.nodejs.org/job/node-test-commit-smartos/247/

I'll move the `setvbuf()` to `main()` or anywhere else that seems more appropriate.